cypress/integration/features/dashboard/dashboard.feature

```diff
@@ -1,14 +1,82 @@
 Feature: To check contents on the dashboard
 
-  As a student I want to be welcomed on the dashboard so that I can start working
+  As a student and a teacher I want to be see important information on the dashboard so that I can be updated start working
 
-  Scenario: Check user greeting on the dashboard
+  Scenario: as a pre-condition teacher creates school news
+    Given I am logged in as a 'teacher' at 'brb'
+    When I go to news overview
+    When I click on add news button
+    Then I see news creation page
+    And I enter news title 'this is a school news'
+    And I enter news description 'test school news description'
+    And I see date input field
+    And I see time input field
+    And I click on Save button
+    Then I see news is created successfully with title 'this is a school news' and with desccription 'test school news description'
+
+  Scenario: as a pre-condition teacher creates a team news
+    Given I am logged in as a 'teacher' at 'brb'
+    When I go to teams overview
+    And I go to a team 'Musik'
+    When I click on news tab on the team detail page
+    And I click on create news button
+    Then I see news creation page
+    And I enter news title 'this is a team news'
+    And I enter news description 'test team news description'
+    And I see date input field
+    And I see time input field
+    And I click on Save button
```

Comment:
"Save button" in capital letters?
In other lines it is not in capital letters (...create news button ..".